tests/resiliency/ledger/tendermint.bats

```diff
@@ -75,3 +75,50 @@ function teardown() {
     check_consistency --pem=1 --balance=999000 --id="$(identity 2)" 8000
     check_consistency --pem=1 --balance=1001 --id="$(identity 3)" 8000
 }
+
+@test "$SUITE: will check duplicated transaction hash in checkTx" {
+    call_ledger --pem=1 --port=8000 send "$(identity 2)" 1000000 MFX
+    call_ledger --pem=1 --port=8000 send "$(identity 3)" 1 MFX
+    check_consistency --pem=1 --balance=1000000 --id="$(identity 2)" 8000
+    check_consistency --pem=1 --balance=1 --id="$(identity 3)" 8000
+
+    # Create a VALID transaction in hexadecimal.
+    msg_hex="$(many message --hex --pem "$(pem 2)" ledger.send "{ 1: \"$(identity 3)\", 2: 1000, 3: \"$MFX_ADDRESS\" }")"
+    echo valid 1: $msg_hex >&3
```

Comment:
You should prepend your messages with a `#` in order to produce 100% TAP-compliant output.

See https://bats-core.readthedocs.io/en/stable/writing-tests.html#printing-to-the-terminal

Note that the `echo` in the test above also needs to be fixed.